hw/ip_templates/alert_handler/dv/cov/alert_handler_cov_excl.el

```diff
@@ -6,110 +6,158 @@
 // This file contains the Excluded objects
 // Generated By User: chencindy
 // Format Version: 2
-// Date: Fri Oct  9 15:24:56 2020
+// Date: Wed Nov 30 12:30:55 2022
 // ExclMode: default
 //==================================================
-CHECKSUM: "1270015083 3167692650"
```

Comment:
For FSM to error state exclusion, no exclusion change, just update the checksum.